docs/get-started/deployment-and-usage.md

```diff
@@ -156,7 +167,53 @@ kubectl kustomize https://github.com/kubernetes-sigs/node-feature-discovery/depl
     kubectl apply -f -
 ```
 
-The example above launces as many jobs as there are non-master nodes. Note that
+The example above launches as many jobs as there are non-master nodes. Note that
+this approach does not guarantee running once on every node. For example,
+tainted, non-ready nodes or some other reasons in Job scheduling may cause some
+node(s) will run extra job instance(s) to satisfy the request.
+
+#### Topologyupdater pod
+
+NFD Topology Updater can be configured along with the default overlay, all
+deployed as separate pods. The `topologyupdater` overlay may be used along
+with `default` overlay to achieve this:
+
+Deploy the `default` overlay:
+
+```bash
+kubectl apply -k https://github.com/kubernetes-sigs/node-feature-discovery/deployment/overlays/default?ref={{ site.release }}
+```
+
+Deploy the `topologyupdater` overlay:
+
+```bash
+kubectl apply -k https://github.com/kubernetes-sigs/node-feature-discovery/deployment/overlays/master-worker-topologyupdater?ref={{ site.release }}
+```
+
+#### Master Worker Topologyupdater
+
+NFD Master, NFD worker and NFD Topology Updater can be configured to be deployed
+as separate pods. The `master-worker-topologyupdater` overlay may be used to
+achieve this:
+
+```bash
+kubectl apply -k https://github.com/kubernetes-sigs/node-feature-discovery/deployment/overlays/master-worker-topologyupdater?ref={{ site.release }}
+
+```
+
+#### Topology Updater One-shot
```

Comment:
nit: only capitalize the first word of section heading (might apply elsewhere, too...)

```suggestion
#### Topologyupdater one-shot
```